v1/edgeloadbalancer/policies_http_types_actions.go

```diff
@@ -166,3 +209,85 @@ func (p *PoliciesHTTPActionLocationRewrite) toVCD() *govcdtypes.AlbVsHttpRespRul
 		KeepQuery: p.KeepQuery,
 	}
 }
+
+// * Helpers to convert PoliciesHTTPActionRateLimit to and from vCD types
+
+func (PoliciesHTTPActionRateLimit) fromVCD(action *govcdtypes.AlbVsHttpSecurityRuleRateLimitAction) *PoliciesHTTPActionRateLimit {
+	if action == nil {
+		return nil
+	}
+	return &PoliciesHTTPActionRateLimit{
+		Count:  action.Count,
+		Period: action.Period,
+		RedirectAction: func() *PoliciesHTTPActionRedirect {
+			if action.RedirectAction != nil {
+				return (&PoliciesHTTPActionRedirect{}).fromVCD(action.RedirectAction)
+			}
+			return nil
+		}(),
+		LocalResponseAction: func() *PoliciesHTTPActionSendResponse {
+			if action.LocalResponseAction != nil {
+				return (&PoliciesHTTPActionSendResponse{}).fromVCD(action.LocalResponseAction)
+			}
+			return nil
+		}(),
+		CloseConnectionAction: func() *bool {
+			if action.CloseConnectionAction == string(PoliciesHTTPConnectionActionCLOSE) {
+				return utils.ToPTR(true)
+			}
+			return nil
+		}(),
+	}
+}
+
+func (p *PoliciesHTTPActionRateLimit) toVCD() *govcdtypes.AlbVsHttpSecurityRuleRateLimitAction {
+	if p == nil {
+		return nil
+	}
+	return &govcdtypes.AlbVsHttpSecurityRuleRateLimitAction{
+		Count:  p.Count,
+		Period: p.Period,
+		CloseConnectionAction: func() string {
+			if p.CloseConnectionAction != nil && *p.CloseConnectionAction {
+				return string(PoliciesHTTPConnectionActionCLOSE)
+			}
+			return ""
+		}(),
+		RedirectAction: func() *govcdtypes.AlbVsHttpRequestRuleRedirectAction {
+			if p.RedirectAction != nil {
+				return p.RedirectAction.toVCD()
+			}
+			return nil
+		}(),
```

Comment:
```suggestion
		RedirectAction: p.RedirectAction.toVCD(),
```

Func toVCD already check if RedirectAction is nil